apps/betterangels-backend/notes/tests/test_queries.py

```diff
@@ -192,57 +192,121 @@ def test_notes_paginated_query(self) -> None:
+        )
+
+        query = """
+            query Notes($filters: NoteFilter) {
+                notes: notesPaginated(filters: $filters) {
+                    totalCount
+                    results{
+                        id
+                    }
+                }
+            }
+        """
+
+        filters: dict[str, Any] = {}
+
+        if case_manager_label:
+            filters["createdBy"] = getattr(self, case_manager_label).pk
+
+        if client_label:
+            filters["client"] = getattr(self, client_label).pk
+
+        if is_submitted is not None:
+            filters["isSubmitted"] = is_submitted
+
+        expected_query_count = 4
+        with self.assertNumQueriesWithoutCache(expected_query_count):
+            response = self.execute_graphql(query, variables={"filters": filters})
+
+        self.assertEqual(response["data"]["notes"]["totalCount"], expected_results_count)
+        notes = response["data"]["notes"]["results"]
+
+        for idx, note_label in enumerate(returned_note_labels):
+            self.assertEqual(notes[idx]["id"], getattr(self, note_label)["id"])
+
+    @parametrize(
+        ("teams, expected_results_count, returned_note_labels, expected_query_count"),
+        [
+            ([SelahTeamEnum.WDI_ON_SITE.name, SelahTeamEnum.SLCC_ON_SITE.name], 2, ["note_2", "note_3"], 4),
+            ([SelahTeamEnum.SLCC_ON_SITE.name], 1, ["note_3"], 4),
+            (["invalid team"], 0, [], 1),
+        ],
+    )
+    def test_notes_query_teams_filter(
+        self,
+        teams: list[SelahTeamEnum],
+        expected_results_count: int,
+        returned_note_labels: list[str],
+        expected_query_count: int,
     ) -> None:
         self.graphql_client.force_login(self.org_1_case_manager_2)
         # self.note is created in the setup block by self.org_1_case_manager_1 for self.client_user_1
         self.note_2 = self._create_note_fixture(
             {
                 "purpose": "Client 1's Note",
-                "title": "Client 1's Note",
-                "publicDetails": "deets",
                 "client": self.client_user_1.pk,
             }
-        )["data"]["createNote"]
-
+        )[
+            "data"
+        ]["createNote"]
         self.note_3 = self._create_note_fixture(
             {
                 "purpose": "Client 2's Note",
-                "title": "Client 2's Note",
-                "publicDetails": "more deets",
                 "client": self.client_user_2.pk,
             }
-        )["data"]["createNote"]
+        )[
+            "data"
+        ]["createNote"]
+        self._update_note_fixture(
+            {
+                "id": self.note_2["id"],
+                "team": SelahTeamEnum.WDI_ON_SITE.name,
+            }
+        )
+        self._update_note_fixture(
+            {
+                "id": self.note_3["id"],
+                "team": SelahTeamEnum.SLCC_ON_SITE.name,
+            }
+        )
+
+        filters = {"teams": teams} if teams else {}
```

Comment:
you don't have a case where teams is falsey, so either add a case or can just move this line down to 323

Reply:
yea, I removed that. Initially, I was going to have a "no filter" testcase but that seems redundant with other tests